bitbots_path_planning:
  ros__parameters:
    base_footprint_frame: base_footprint
    tf_buffer_duration: 5.0
    rate: 20.0
    map:
      planning_frame: map
      resolution: 20
      size:
        x: 11.0
        y: 8.0
    controller:
      carrot_distance: 20
      rotation_accuracy: 0.1
      position_accuracy: 0.05
      max_rotation_vel: 0.4
      max_vel_x: 0.25
      min_vel_x: -0.2
      max_vel_y: 0.08
      smoothing_k: 0.4
      rotation_slow_down_factor: 0.3
      translation_slow_down_factor: 0.5
      orient_to_goal_distance: 1.0
